tests/testthat/test-utils.R

```diff
@@ -1,4 +1,5 @@
 ref_lkup <- fst::read_fst("../testdata/app_data/20210401/estimations/interpolated_means.fst")
+ref_lkup <- data.table::as.data.table(ref_lkup)
```

Comment:
This is one of the files which @randrescastaneda was not able to find. Reported here #281 . Can you clarify with him since I am not able to check it from my end. 

Also, `fst::read_fst` has `as.data.table` argument when set to `TRUE` would return the data.table object.